### PR TITLE
Add Pydio. Closes #335

### LINKED ISSUE #335: App Suggestion: Pydio-cells
**Please tell us what app you'd like to see on CapRover as a one-click app.**

Pydio, it's a great alternative to nextcloud

**Do you know if there is any official Docker image for the app?**

yes, https://hub.docker.com/r/pydio/cells 

There is a docker-compose example, but the install process is a pain and I couldn't get it to work

### PR DESCRIPTION
**Pull request by kgnfth**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by kgnfth** [comment]
Why is the `Closes` thing not working ☹️

File: public/v4/apps/pydio.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            4. Click on Edit Nginx Configuration and right under
+
+            "set $upstream https://<%-s.localDomain%>:<%-s.containerHttpPort%>;" add the following code
+
+                location /ws/ {
+                    proxy_pass $upstream;
+                    proxy_http_version 1.1;
+                    proxy_set_header Upgrade $http_upgrade;
+                    proxy_set_header Connection "Upgrade";
+                }
```
Comment (githubsaturn) [inline public/v4/apps/pydio.yml:72]: Why is this block needed? Enabling websocket from the UI does the same thing.

Comment (kgnfth) [reply public/v4/apps/pydio.yml:72]: i enabled the websocket but it gave me the following error in my browser console
```
VM1413:91 WebSocket connection to 'wss://pydio.app.gocloud.fun/ws/event' failed: 
connect @ VM1413:91
VM1413:118 WebSocket Closed Connection:The connection was closed abnormally, e.g., without sending or receiving a Close control frame (code 1006)
```
When i added that code, the error was gone

i have the instructions from https://pydio.com/en/docs/kb/deployment/running-cells-behind-nginx-reverse-proxy

Comment (githubsaturn) [reply public/v4/apps/pydio.yml:72]: This is really weird... Every single line of instruction here is enabled by CapRover... not sure what that wouldn't work.

https://github.com/caprover/caprover/blob/master/template/server-block-conf.ejs#L58-L84

**Pull request merged**